### PR TITLE
chore: add mkdocs check

### PR DESCRIPTION
**Pull request by whynowy**
1. Add mkdocs check;
2. Upgrade go version
3. Deprecate `ioutil`

### CODE REVIEW

File: pkg/sources/generator/tickgen.go [hunk 1]
```diff
@@ -114,7 +114,9 @@ func WithReadTimeOut(timeout time.Duration) Option {
 
 // NewMemGen fuction creates an instance of generator.
 // ctx  - context passed by the cmd/start.go a new context with cancel
-//        is created for use by this vertex.
+//
```
Comment (vigith) [inline pkg/sources/generator/tickgen.go:117]: is this intended? i think that comment was part of prev line

File: pkg/watermark/processor/entity.go [hunk 2]
```diff
@@ -57,7 +57,8 @@ var _ ProcessorEntitier = (*ProcessorEntity)(nil)
 
 // _defaultKeySeparator is the key separate when we have shared OT buckets.
 // NOTE: we can only use `_` as the separator, Jetstream will not let any other special character.
-//       Perhaps we can encode the key using base64, but it will have a performance hit.
+//
```
Comment (vigith) [inline pkg/watermark/processor/entity.go:60]: I think this is also unintended. It does look like a block code comment.

**Review by vigith — APPROVED** [review]

File: pkg/sources/generator/tickgen.go (hunk 1 above)
Comment (whynowy) [reply pkg/sources/generator/tickgen.go:117]: It's go 1.19 format.

File: pkg/watermark/processor/entity.go (hunk 2 above)
Comment (whynowy) [reply pkg/watermark/processor/entity.go:60]: Same here, 1.19 format.

**Pull request merged**